backend/audit/models.py

```diff
@@ -160,6 +162,40 @@ def save(self, *args, **kwargs):
 
         return super().save()
 
+    def disseminate(self):
+        """
+        Cognizant/Oversight agency assignment followed by dissemination
+        ETL.
+        """
+        # try:
+        self.assign_cog_over()
+        intake_to_dissem = IntakeToDissemination(self)
+        intake_to_dissem.load_all()
+        intake_to_dissem.save_dissemination_objects()
+        # TODO: figure out what exceptions to catch here
+        # except Exception as err:
+        #     return {"error": err}
+
+    def assign_cog_over(self):
```

Comment:
I don't see why it's better to move this logic here. But it should work for now since we won't have to deal with overrides forat least another month. Hope you looked into the case where a Cog Assignment is manually overridden  - we then need to update SAC and Dissemination then as well.